build

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+#yarn has to be installed first
+#
+npm run-script build
```

Comment:
There are some unrelated files.

Reply:
Helper file to build optional but I submitted an issue about github forking not complete where I completed this script to be able to build editorjs. see https://github.com/codex-team/editor.js/issues/1283

Reply:
I read https://github.com/codex-team/editor.js/issues/1283, and commented. 
Editor.js uses git sub modules. 
https://github.com/codex-team/editor.js/issues/1283#issuecomment-680714370